README.md

```diff
@@ -213,6 +213,22 @@ client.authorizeCustomer({
 
 If both the client config and the function call define the same fetch option with different values, the fetch option value for the function call will take priority. In the examples above, both pass in the `redirect` fetch option with different values, however, `redirect: "manual"` will take precedence because it was passed on the function call level. 
 
+### Custom Query Parameters
+
+With the introduction of [hooks for Commerce APIs](https://developer.salesforce.com/docs/commerce/commerce-api/guide/extensibility_via_hooks.html), customers can pass custom query parameters through the SDK to be used in their custom hook. Custom query parameters must begin with `c_`:
+
+```javascript
+const searchResults = await searchClient.productSearch({
+  parameters: { 
+    q: "dress", 
+    limit: 5,
+    c_customQueryParam: '<your-custom-query-param>'
```

Comment:
Nit picky but if we want to be consistent with the naming with respect to the data object those names represent we might want something like this:

```
c_paramkey: '<param-value>'
```

the "custom" in `c_customQueryParam` is a little redundant because thats what "c" stands for, and "custom" in `'<your-custom-query-param>'` is a misnomer because all values are kind custom.